### DISCUSSION TITLE
Vercel Design System

### DISCUSSION THREAD

**Discussion by BuchananQuantum** [heroui-inc/heroui#648] [General]
From working with NextUI and seeing the relationship to the Vercel Design System (@geist-ui/core). I'm wondering if this if their official open source release.

**Comment by jrgarciadev** [comment]
Hey @BuchananQuantum NextUI is not a Vercel project and the design system is more inspired by [Vuesax](https://vuesax.com/) & [Human Interface](https://developer.apple.com/design/human-interface-guidelines/guidelines/overview) than in the Vercel design system
